mongodbatlas/online_archives.go

```diff
@@ -192,12 +192,24 @@ type OnlineArchive struct {
 	ClusterName     string                 `json:"clusterName,omitempty"`
 	CollName        string                 `json:"collName,omitempty"`
 	CollectionType  string                 `json:"collectionType,omitempty"`
-	Criteria        *OnlineArchiveCriteria `json:"criteria,omitempty"`
 	DBName          string                 `json:"dbName,omitempty"`
 	GroupID         string                 `json:"groupId,omitempty"`
-	PartitionFields []*PartitionFields     `json:"partitionFields,omitempty"`
-	Paused          *bool                  `json:"paused,omitempty"`
 	State           string                 `json:"state,omitempty"`
+	Paused          *bool                  `json:"paused,omitempty"`
+	Criteria        *OnlineArchiveCriteria `json:"criteria,omitempty"`
+	Schedule        *OnlineArchiveSchedule `json:"schedule,omitempty"`
+	PartitionFields []*PartitionFields     `json:"partitionFields,omitempty"`
```

Comment:
can you keep the fields alphabetically sorted?

Reply:
this order should be more memory efficient. Do you have a strong opinion here? If that is the case I will revert it. Thanks!

Reply:
>this order should be more memory efficient

Have we enabled that linter? Personally I like alphabetical order of props as it gives you a quick way to verify if a field you are looking for exists but no strong opinion to rollback.
Now on memory efficiency I don't think these are that big and this may fall into a bit of micro improvement/premature optimization but if you want to do it I think you should enable that linter

Reply:
> Have we enabled that linter

no, we haven't.

> Personally I like alphabetical order of props as it gives you a quick way to verify if a field you are looking for exists but no strong opinion to rollback.
> Now on memory efficiency I don't think these are that big and this may fall into a bit of micro improvement/premature optimization but if you want to do it I think you should enable that linter

Make sense. I don't think makes sense to enable the linter at this point since we will move to the autogenerated sdk. I will revert it. Thanks!

Reply:
Updated